src/GTAQuat.cpp

```diff
@@ -57,7 +54,7 @@ vec3 GTAQuat::ToEuler() const
 		rz = -degrees(atan2(clamp(x * y + z * w, -1.0f, 1.0f), clamp(0.5f - x * x - z * z, -1.0f, 1.0f)));
 	}
 
-	// Keep each component in the [0.0f, 360.0f[ interval
+	// Keep each component in the [0.0f, 360.0f] interval
```

Comment:
`[` means not included in the interval

Reply:
Oh, in that case it should be `)`

Reply:
Ah well, wikipedia mentions [both](https://en.wikipedia.org/wiki/Interval_(mathematics)#Notations_for_intervals), just more used to `[` in my country.

Reply:
No worries, it's just standard math notation to use brackets for inclusive and parenthesis for non-inclusive from what I ran into here, but it seems both notations are accepted. In that case there's no real issue and can just stay as it is, thanks for the info.